ElastosBountyProgram/front-end/src/config/router.js

```diff
@@ -399,7 +399,7 @@ export default [
     },
     {
         path: '/admin/teams/:teamId',
-        page: AdminTeamDetailPage
+        page: TeamDetailPage
```

Comment:
You should adjust the AdminTeamDetailPage, otherwise we lose the admin breadcrumb and navigation.